src/lib/icons/Check.svelte

```diff
@@ -1,9 +1,18 @@
+<script lang="ts">
```

Comment:
This seems overly verbose for replacing `$$restProps`, there could be a better way.

Reply:
Updated in https://github.com/oddbird/oddcontrast/commit/a79e99a23789e50bc5b565788cbcc5a20fecabf5